forms-html/zainab_dunia/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+</head>
+
+<body>
+  <form action="">
+    <label for="email">Enter your Email:</label><br>
+    <input type="email" id="email"><br>
+    <label for="name">Username:</label><br>
+    <input type="text" id="name"><br>
+    <label for="pass">Enter your password:</label><br>
+    <input type="password" id="pass"><br>
+    <label for="num">Quantity(between 1 and 5):</label>
+    <input type="number" id="num"><br>
```

Comment:
To restrict the user to only enter number between 1 and 5 you can use the min and max attributes to enforce these restrictions.